erpnext/patches/v9_2/rename_net_weight_in_item_master.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import unicode_literals
+import frappe
+from frappe.model.utils.rename_field import rename_field
+
+def execute():
+	frappe.reload_doc("stock", "doctype", "item")
+	if frappe.db.has_column('Item', 'net_weight'):
+		rename_field("Item", "net_weight", "weight_per_unit")
```

Comment:
Patch will break if field weight_per_unit is exists!

Reply:
if exists copy data from net_weight to weight_per_unit else rename_field("Item", "net_weight", "weight_per_unit")